src/Orleans.Core/Configuration/Options/ClusterOptions.cs

```diff
@@ -21,12 +31,12 @@ public class ClusterOptions
         /// <summary>
         /// Gets or sets the cluster identity. This used to be called DeploymentId before Orleans 2.0 name.
         /// </summary>
-        public string ClusterId { get; set; }
+        public string ClusterId { get; set; } = DefaultClusterId;
 
         /// <summary>
         /// Gets or sets a unique identifier for this service, which should survive deployment and redeployment, where as <see cref="ClusterId"/> might not.
         /// </summary>
-        public string ServiceId { get; set; }
+        public string ServiceId { get; set; } = DevelopmentClusterId;
```

Comment:
I think you mean to write DefaultServiceId here.

Reply:
Thanks, will fix